options/internal/generic/essential.cpp

```diff
@@ -13,6 +13,7 @@ namespace {
 
 	template<typename T>
 	[[gnu::always_inline]]
+	[[gnu::no_sanitize("all")]] // Since we use unaligned loads.
```

Comment:
Why are unaligned loads a problem? `word<T>` should have an alignment of 1.

Reply:
We discussed this a bit on Discord. `word<T>` does not have an alignment of 1 despite the attribute. I don't think there's an obvious alternative, so I'm inclined to keep the `no_sanitize` for now.